core/src/main/java/net/opentsdb/query/ReadCacheQueryPipelineContext.java

```diff
@@ -917,30 +966,51 @@ protected boolean checkComplete() {
           (context.query().getCacheMode() == CacheMode.NORMAL ||
            context.query().getCacheMode() == CacheMode.WRITEONLY)) {
         if (full_query_context.cacheable()) {
+          full_query_caching.set(true);
```

Comment:
What consumes this new field?

Reply:
actually.... nothing, need to remove it.